ChatRPG/ChatRPG/Properties/launchSettings.json

```diff
@@ -23,7 +34,8 @@
       "launchBrowser": true,
       "applicationUrl": "https://localhost:7014;http://localhost:5111",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "UseMocks": "true"
```

Comment:
Jeg vil foretrække at det er noget man dynamisk kan ændre i applikationen, f.eks. ved at trykke på en checkbox. Det kan vi genbesøge når det bliver relevant i projektet

Reply:
Det kan vi også sagtens lave, så skal vores api client bare instantiere to clients, og så vælger den en i kaldet baseret på bool (her checkbox).

Noget vi ikke har snakket om er om vi skal lave en LlmService? Denne er jo specifik til openai, og der skal laves nyt interface til f.eks. Llama - men hvis vi laver en service kan vi skifte api client ud, så længe den indeholder en metode der tager imod input og kan producere et output

Reply:
Enig, det er lidt det jeg ville hen imod med min kommentar til `IOpenAiClient`. Men jeg synes stadig det er okay at bygge efter Open AIs API reference, for det er muligt at lave en API der severer stort set alle andre LLMs i det format (llama har endda et lille eksempel på en API der fungerer på samme måde som ChatGPTs)